angular/projects/admin-nrpti/src/app/mines/mines-collection-detail/mines-collection-detail-record-row/mines-collection-detail-record-row.component.html

```diff
@@ -0,0 +1,14 @@
+<td scope="row" data-label="Name" class="col-6">{{ rowData.name || '-' }}</td>
+<td scope="row" data-label="Source System" class="col-2">{{ rowData._sourceSystem || '-' }}</td>
```

Comment:
I don't think `_sourceSystem` exists, not with that specific variable name that is.

Reply:
You are right, it is `sourceSystemRef` in all models. Updated.